sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - `RecognizeContentOperation` now returns a `FormPageCollection`.
 - `RecognizeReceiptsOperation` now returns a `RecognizedReceiptCollection`.
 - `RecognizeCustomFormsOperation` now returns a `RecognizedFormCollection`.
+- `FieldValue.AsInt32` replaced by `FieldValue.AsInt64`, which returns a `long` value.
```

Comment:
Is it worth calling out in a few words that this is because of a change to the service?

Reply:
Updated it.